### ISSUE TITLE
Doesn't support texture modifiers

### ISSUE THREAD

**Issue by ThePython10110** [minetest-mods/3d_armor#112]
Both the "preview" images and the actual textures do not support texture modifiers, just the inventory images.

**Comment by BuckarooBanzay** [comment]
That's correct iirc, pr's welcome, but the code might need some cleanup first :/
